assets/base.css

```diff
@@ -2957,3 +2957,41 @@ details-disclosure > details {
     outline: transparent solid 1px;
   }
 }
+
+/* Animations */
+
+.animate {
+  transition: opacity 0.55s ease-in-out;
```

Comment:
You can experiment with the `cubic-bezier` easing function instead of `ease-in-out` if you want more control. This web app is a neat tool I just found: https://cubic-bezier.com/#.17,.51,.31,1